routes/.eslintrc.json

```diff
@@ -0,0 +1,6 @@
+{
+  "rules": {
+    "no-param-reassign": "off"
```

Comment:
Ok. If we add a `sum` to each keyword then we can use findByIdAndUpdate with $push when updating the keywords too.